cocoa/BugsnagReactNative.m

```diff
@@ -265,8 +266,14 @@ + (void)startWithConfiguration:(BugsnagConfiguration *)config {
     config.shouldAutoCaptureSessions = [RCTConvert BOOL:options[@"autoCaptureSessions"]];
     config.automaticallyCollectBreadcrumbs = [RCTConvert BOOL:options[@"automaticallyCollectBreadcrumbs"]];
 
+    [config addBeforeSendSession:^void(NSMutableDictionary *_Nonnull data) {
+        data[@"device"] = [self addDeviceRuntimeVersion:data[@"device"]];
+    }];
+
     [config addBeforeSendBlock:^bool(NSDictionary *_Nonnull rawEventData,
                                      BugsnagCrashReport *_Nonnull report) {
+        report.device = [self addDeviceRuntimeVersion:report.device];
```

Comment:
This will append the runtime version of the app when sending the report, which isn't necessarily the same as when capturing the crash.